ios/SegmentAnalytics/Classes/SegmentAnalytics.m

```diff
@@ -14,7 +14,9 @@ @implementation SegmentAnalytics
 RCT_EXPORT_METHOD(setup:(NSString*)configKey) {
     SEGAnalyticsConfiguration *configuration = [SEGAnalyticsConfiguration configurationWithWriteKey:configKey];
     configuration.flushAt = 1;
+    configuration.recordScreenViews = YES;
     configuration.shouldUseLocationServices = true;
+    configuration.trackApplicationLifecycleEvents = YES;
```

Comment:
The style of booleans used in this method is `true` (see line 18), did you consider using that for your additions?